src/components/Header/Header.tsx

```diff
@@ -0,0 +1,11 @@
+import Title from "./Header.styled"
+
+const Header = () => {
+    return(
+        <Title>
+            
+        </Title>
```

Comment:
```suggestion
        <Title />
```

This should be a self closing tag